sormas-base/setup/s2s-import-to-truststore.sh

```diff
@@ -23,11 +23,11 @@ echo "# SORMAS TO SORMAS CERTIFICATE IMPORT"
 echo "# This script imports a certificate into the local truststore, to be used for SORMAS2SORMAS communication"
 echo "# If anything goes wrong, please consult the sormas to sormas import guide or get in touch with the developers."
 
-if [[ $(expr substr "$(uname -a)" 1 5) = "Linux" ]]; then
-	LINUX=true
-else
-	LINUX=false
-fi
+#if [[ $(expr substr "$(uname -a)" 1 5) = "Linux" ]]; then
+LINUX=true
+#else
+#	LINUX=false
+#fi
```

Comment:
Why is this commented out?

Reply:
I'm working on mac so this condition evaluates LINUX to be false which and the script doesn't work.

The easiest workaround was to comment it but I forgot to uncomment before push :)